python_docs_theme/__init__.py

```diff
@@ -16,16 +16,24 @@
 MESSAGE_CATALOG_NAME = "python-docs-theme"
 
 
+def add_translation_to_context(app, pagename, templatename, context, doctree):
+    theme_gettext = get_translation(MESSAGE_CATALOG_NAME)
+    sphinx_gettext = get_translation("sphinx")
+
+    def combined(msg):
+        trans = theme_gettext(msg)
+        if trans == msg:
+            return sphinx_gettext(msg)
```

Comment:
Do we have strings that we don't extract, or rather it is used by languages that doesn't have theme translation and subset of strings is available in Sphinx translation?